src/hotspot/share/utilities/resourceHash.hpp

```diff
@@ -202,14 +223,15 @@ class ResourceHashtableBase : public STORAGE {
     return &(*ptr)->_value;
   }
 
-
-  bool remove(K const& key) {
+  template<typename Function>
+  bool remove(K const& key, Function function) {
     unsigned hv = HASH(key);
     Node** ptr = lookup_node(hv, key);
 
     Node* node = *ptr;
     if (node != nullptr) {
       *ptr = node->_next;
+      bool cont = function(node->_key, node->_value);
```

Comment:
Q: The local `cont` is not used. Just wanted to check if anything is missed here.
     Also, what does this name mean? Should it be named `cond` instead?

Reply:
The 'cont' variable was because I cut/pasted the lambda from iterate and in that case means to continue.  That's also not needed for 'remove' so I removed the return variable for the lambda function.